src/Cli.Tests/AddEntityTests.cs

```diff
@@ -67,6 +69,8 @@ public Task AddNewEntityWhenEntitiesNotEmpty()
                 fieldsToExclude: new string[] { },
                 policyRequest: null,
                 policyDatabase: null,
+                cacheEnabled: null,
```

Comment:
We should add automated tests that test if the these cache option get added or not using the flags in the add command.